python-package/xgboost/dask.py

```diff
@@ -1223,61 +1264,21 @@ class DaskScikitLearnBase(XGBModel):
 
     # pylint: disable=arguments-differ
     @_deprecate_positional_args
-    def fit(
-        self,
-        X: _DaskCollection,
-        y: _DaskCollection,
-        *,
-        sample_weight: Optional[_DaskCollection] = None,
-        base_margin: Optional[_DaskCollection] = None,
-        eval_set: List[Tuple[_DaskCollection, _DaskCollection]] = None,
-        eval_metric: Optional[Callable] = None,
-        sample_weight_eval_set: Optional[List[_DaskCollection]] = None,
-        early_stopping_rounds: Optional[int] = None,
-        verbose: bool = True,
-        feature_weights: Optional[_DaskCollection] = None,
-        callbacks: List[TrainingCallback] = None
-    ) -> "DaskScikitLearnBase":
```

Comment:
Why are we removing the docstring for `fit()`?

Reply:
The doc string comes from inheritance as now we support all parameters.